store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1090,7 +1090,7 @@ private boolean execute(String args[]) throws ServiceException, ArgException, IO
         }
         switch (command) {
         case ADD_ACCOUNT_ALIAS:
-            prov.addAlias(lookupAccount(args[1]), args[2]);
+            prov.addAlias(lookupAccount(args[1]), args[2], false);
```

Comment:
Is it okay to always set the 3rd param to `false`? Can't an administrator choose  `true` or `false` in zmprov command?

Reply:
In the current ticket, we didn't work on the CLI commands. This will be enhanced in the next ticket. Made a note of it already

Reply:
rather than come back and modify this code again, you should make actual implementation.